Tests/configure_and_test_integration_instances.py

```diff
@@ -1296,13 +1297,79 @@ def set_marketplace_url(servers, branch_name, ci_build_number):
     sleep(60)
 
 
-def main():
-    install_logging('Install Content And Configure Integrations On Server.log')
-    build = Build(options_handler())
+def test_integrations_post_update(build: Build, new_module_instances: list, old_module_instances: list) -> tuple:
+    """
+    Runs 'test-module on all integrations for post-update check
+    Args:
+        build: A build object
+        new_module_instances: A list containing new integrations instances to run test-module on
+        old_module_instances: A list containing old (existing) integrations instances to run test-module on
+
+    Returns:
+        * A list of integration names that have failed the 'test-module' execution post update
+        * A list of integration names that have succeeded the 'test-module' execution post update
+    """
+    old_module_instances.extend(new_module_instances)
+    successful_tests_post, failed_tests_post = instance_testing(build, old_module_instances, pre_update=False)
+    return failed_tests_post, successful_tests_post
+
+
+def update_content_on_servers(build: Build, installed_content_packs_successfully: bool) -> bool:
+    """
+    Updates content on the build's server according to the server version
+    Args:
+        build: Build object
+        installed_content_packs_successfully: whether the content packs were installed successfully pre-update
+
+    Returns:
+        A boolean that indicates whether the content installation was successful.
+        If the server version is lower then 5.9.9 will return the 'installed_content_packs_successfully' parameter as is
+        If the server version is higher or equal to 6.0 - will return True if the packs installation was successful
+        both before that update and after the update.
+    """
+    if LooseVersion(build.server_numeric_version) < LooseVersion('6.0.0'):
+        update_content_till_v6(build)
+    elif not build.is_nightly:
+        set_marketplace_url(build.servers, build.branch_name, build.ci_build_number)
+        installed_content_packs_successfully = install_packs(build) and installed_content_packs_successfully
+    return installed_content_packs_successfully
+
+
+def configure_and_test_integrations_pre_update(build: Build, new_integrations, modified_integrations) -> tuple:
+    """
+    Configures integration instances that exist in the current version and for each integration runs 'test-module'.
+    Args:
+        build: Build object
+        new_integrations: A list containing new integrations names
+        modified_integrations: A list containing modified integrations names
+
+    Returns:
+        A tuple consists of:
+        * A list of old module instances configured
+        * A list of new module instances configured
```

Comment:
what are `old` and `new` means?

Reply:
Old means integrations that already exist 
New means integrations that were added in this PR

Reply:
then let's use new and modified